test/utils/duration_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+
+import 'package:lichess_mobile/src/utils/duration.dart';
+
+void main() {
+  group('DurationExtensions.toDaysHoursMinutes()', () {
+    // The following tests are widget tests because the function being
+    // tested need access to the BuildContext, and this is a simple way to access it.
```

Comment:
Not a big fan of doing like that. I see a couple of options:
- instead of passing `BuildContext` as arg to the function you can pass `AppLocalization` which can be easily mocked 
- instead of an extension you can use a service with `Riverpod` that depends on the `l10nProvider`: https://github.com/lichess-org/mobile/blob/main/lib/src/common/l10n.dart#L7 (that way this method will be accessible anywhere, as long there is a `ref` in the scope, no need for the `BuildContext`.